slo-workload/src/Commands/RunCommand.php

```diff
@@ -323,6 +327,16 @@ protected function metricsJob(int $reportPeriod, float $startTime, int $time, st
         }
     }
 
+    protected function getLatency($begin)
```

Comment:
Need units in function name, for example: getLanencySeconds
and comment with return type: int/float